src/Security/Authentication/OpenIdConnect/samples/OpenIdConnectSample/Startup.cs

```diff
@@ -32,14 +32,36 @@ public Startup(IConfiguration config, IWebHostEnvironment env)
         public IConfiguration Configuration { get; set; }
         public IWebHostEnvironment Environment { get; }
 
+        private void CheckSameSite(HttpContext httpContext, CookieOptions options)
+        {
+            if (options.SameSite > SameSiteMode.Unspecified)
+            {
+                var userAgent = httpContext.Request.Headers["User-Agent"];
```

Comment:
`var userAgent = httpContext.Request.Headers["User-Agent"].ToString();`

Reply:
Do you need to make this change in the preview1 build?

Reply:
No, it's just a sample, I've corrected it elsewhere. I just wanted to mark the typo so I'd remember it.